README.md

```diff
@@ -103,6 +103,7 @@ Awesome software projects sub-categorized by focus.
 - [OpendTect](https://dgbes.com/index.php/software#free) – Seismic interpretation package
 - [RINGMesh](https://github.com/ringmesh/RINGMesh) – ![C++](media/icon/cplusplus.png) Mesh manipulation of geological models
 - [QGIS](http://www.qgis.com/) – GIS platform to visualize, manage, edit, analyse data, and compose printable maps
+- [Pangeo](https://pangeo.io/) – ![Python](media/icon/python.png) A community platform for Big Data geoscience
```

Comment:
This could read more like: 

"Pangeo: A community platform for Big Data geoscience built on top of the open source scientific python ecosystem"